### PR TITLE
[21] Contact Form Validation

### PR DESCRIPTION
**Pull request by Oyeteju**
## Task: `Validate the Contact form`
- `A customer user must be able to fill a contact form so he can reach out to the e-commerce store Admin.`

### CODE REVIEW

File: submissions/oyetejuOloyede/DOM/contactForm.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          return false;
+          }
+      else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+      }
+}
+
+function validation(e) {
+    valid(/^[a-z]+$/i, fullName, fullNameMsg, fullNameMsg, e);
```
Comment (adaugochi) [inline submissions/oyetejuOloyede/DOM/contactForm.js:29]: `error1` and `error2` have the same error message. is it suppose to be so?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oyetejuOloyede/DOM/contactForm.js (hunk 1 above)
Comment (Oyeteju) [reply submissions/oyetejuOloyede/DOM/contactForm.js]: error1 and error2 corrected

**Review by adaugochi — CHANGES_REQUESTED** [review]
You didn't push anything

**Comment by ToluAkin** [comment]
@Oyeteju The pull request is empty. Check [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/748/files)

**Comment by adaugochi** [comment]
Long over due

**Pull request closed without merging**